### PR TITLE
add Tukey's biweight function for anisotropic_diffusion

### PR DESCRIPTION
**Pull request by cancan101**
Closes https://github.com/loli/medpy/issues/51

Using:
image
from: http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.449.948&rep=rep1&type=pdf

### CODE REVIEW

File: medpy/filter/smoothing.py [hunk 1]
```diff
@@ -123,6 +130,12 @@ def condgradient(delta, spacing):
     elif option == 2:
         def condgradient(delta, spacing):
             return 1./(1.+(delta/kappa)**2.)/float(spacing)
+    elif option == 3:
+        kappa_s = kappa * (2**0.5)
+
+        def condgradient(delta, spacing):
+            top = 0.5/((1.-(delta/kappa_s)**2.)**2.)/float(spacing)
+            return numpy.where(delta <= kappa_s, top, 0)
```
Comment (cancan101) [inline medpy/filter/smoothing.py]: I think there should be an abs on the delta here?

Comment (loli) [reply medpy/filter/smoothing.py]: I would think so, too. The problem didn't arise with the Perona-Malik diffusion equations and the Tsiotsios et al. work does not address the problem. Could you fix this? Then I'll approve the pull.

Thanks a lot for the addition!

Comment (cancan101) [reply medpy/filter/smoothing.py]: [This](https://github.com/shirley9073/mipav/blob/3716b62b4715718424f5be92d10e70876822c0aa/src/gov/nih/mipav/model/algorithms/AlgorithmFastMarching.java#L27) seems to indicate using the abs

**Comment by cancan101** [comment]
Here are the flow functions (kappa=1):
image

**Comment by loli** [comment]
Sweet. It would be wonderful if this could go into the functions documentation string... I remember vaguely that Sphinx supports images.

```
   .. image:: _static/my_image_small.png
```

The question would be where to put it.

**Pull request merged**